app/models/concerns/edm/entity/referenced_records.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module EDM
+  module Entity
+    module ReferencedRecords
+      include RepositoryHelper
+      extend ActiveSupport::Concern
+
+      def unreferenced?
```

Comment:
Given the above comment, this would need to become a method on the view class instead of on the model.